### ISSUE TITLE
Display Calendar Colors as side indicators

### ISSUE THREAD

**Issue by stefan-niedermann** [owncloud/calendar#874]
Hi guys,

what do you think about displaying the calendar colors as side indicators like in Tasks App? I think this would not look as heavy as the current squares.

**current**
![current](https://cloud.githubusercontent.com/assets/4741199/8801521/ae663caa-2fbb-11e5-9504-8e3004cad70c.png)

**tasks-app**
![expected](https://cloud.githubusercontent.com/assets/4741199/8801529/bb2f5764-2fbb-11e5-807f-b444abbcb4f2.png)

cc @owncloud/designers @jancborchardt

**Comment by MorrisJobke** [comment]
Nice idea :)

**Comment by jancborchardt** [comment]
Good point to unify the look of the apps! However, the squares have a function in the Calendar as they are checkboxes to show/hide the respective calendar. Hence we shouldn’t make them bars because then they are not clickable.

Rather, I would put the Tasks group color indicator bar on the left, next to the group name. On the right they are next to the number only and you need to move your eye quite far to connect color and name. cc @Henni @raimund-schluessler

**Comment by raimund-schluessler** [comment]
I also like the idea of making it a bar instead of a square. The calendar visibility is currently not only toggled by the square, you can also just click somewhere on the entry (e.g. the title), so keeping it a square is not really necessary. (Just a side note: I would expect a colorpicker to choose the calendar color when clicking on the square, not toggling the visibility, but that is just my point of view ;).

Of course we can move the bar to the left for tasks.

**Comment by jancborchardt** [comment]
> I would expect a colorpicker to choose the calendar color when clicking on the square, not toggling the visibility

Good point actually, I also thought that. So maybe we should indeed make it a bar on the side only to not mislead. Changing color will be possible in the edit mode anyway. @libasys what do you think about that for Calendar+?

**Comment by libasys** [comment]
Not really good cause the colored square on the leftside is needed for activating the default calendar, means if you create an event the default calendar gets choosen!

**Comment by jancborchardt** [comment]
@libasys since there is confusion as to what the colored square does, choosing the default calendar could still be done by a checkbox left of the calendar, only not colored. Or in the dot-dot-dot details menu.
